src/queue/PlaylistState.cxx

```diff
@@ -94,7 +94,7 @@ playlist_state_save(BufferedOutputStream &os, const struct playlist &playlist,
 	os.Fmt(FMT_STRING(PLAYLIST_STATE_FILE_SINGLE "{}\n"),
 			  (int)playlist.queue.single);
 	os.Fmt(FMT_STRING(PLAYLIST_STATE_FILE_CONSUME "{}\n"),
-	       (unsigned)playlist.queue.consume);
+	       	  (int)playlist.queue.consume);
```

Comment:
What's the point of changing this to `signed`, when the base type of your enum is `uint8_t`?

Reply:
I copied it from playlist.queue.single. It is in both cases unnecessary?

Reply:
Yes. There's not only no negative enum value, but those enums are explicitly declared to be unsigned. Casting them to signed makes no sense.